website/docs/extensions/publish/index.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 2
+title: Publishing
+description: Publish Podman Desktop extension
+tags: [podman-desktop, extension, publishing]
+keywords: [podman desktop, extension, publishing]
+---
+
+#### Prerequisites
+
+* [Writing Extension](../extensions/write)
+
+Consider publishing and distributing as extension builds successfully.
+
+Extension consists of a zip file of all javascript files, dependencies and metadata.
+
+Extension assembly consists of a .cdix file or an OCI image.
+
+Publishing to an OCI registry is the recommended way for the publication.
+
+
+### Creating the OCI image and publish image
+
+Considers building an OCI image to publish to quay.io, dockerhub or other OCI registries
+
+#### Containerfile example
+
+1. Use a scratch image as it only requires the content of the assembly, no runtimeulti-
```

Comment:
runtimeulti- ? Is this a typo, or just some terminology I'm not familiar with?

Reply:
it's when you type in a bus :-)

Reply:
fixed, thanks